ui/Makefile

```diff
@@ -10,7 +10,7 @@ ROX_PRODUCT_BRANDING ?= $(shell $(MAKE) --quiet --no-print-directory -C .. produ
 export REACT_APP_ROX_PRODUCT_BRANDING := $(ROX_PRODUCT_BRANDING)
 
 deps: apps/platform/package.json apps/platform/package-lock.json
-	cd apps/platform && npm ci --prefer-offline --fetch-timeout=$(UI_PKG_INSTALL_NETWORK_TIMEOUT) --no-fund
+	cd apps/platform && npm ci --prefer-offline --fetch-timeout=$(UI_PKG_INSTALL_NETWORK_TIMEOUT) --no-fund --no-audit $(UI_PKG_INSTALL_EXTRA_ARGS)
```

Comment:
What is the reason to additionally add the `--no-audit` flag?

Reply:
This was from my testing to try to determine what was causing the OOM errors post-install, and I left it as a way to reduce additional output since we are not interested in potential vulnerabilities in the output during CI jobs.

However, I've recently switched to running `make deps` myself to install dependencies, so I'll re-add this in order to see the vuln report locally.